docs/Reference/API-Methods.md

```diff
@@ -4489,6 +4489,127 @@ one object per transaction, in transaction execution order.
         ]
     }
     ```
+### trace_transaction
+
+Provides transaction processing of [type `trace`](Trace-Types.md#trace) for the specified transction.
+
+!!! important
+
+    Your node must be an archive node (that is, synchronised without pruning or fast sync) or the
+    requested block must be within [the number of pruning blocks retained](../CLI/CLI-Syntax#pruning-blocks-retained).
```

Comment:
Line length?

Reply:
It's a link putting it over so I'm going to leave it in this case